### ISSUE TITLE
Support of OptiFine_1.7.10_HD_U_E7 (old MC version but with powerfull modpacks)

### ISSUE THREAD

**Issue by Ngar1** [jrbudda/minecrift#348]
Vivecraft supports only the very old Optifine version "D1" which is since longer time incompatibe with serveral mods.
Ideal would be the support of OptiFine_1.7.10_HD_U_E7, or as minimum OptiFine_1.7.10_HD_U_D6.

I play GregTechNewHorizons Modpack with very active modding community, but they cant help to support such old Optifine version.
This modpack is supported since 7 years, with many new release and new (& exclusive) mods per year. 1.7.10 is "old", but with newest mods newer MC version is not beats it.

Is there something to motivate this update?

**Comment by longlongCODE** [comment]
收到邮件了，但是如果邮件很重要还请你发一条短信通知我收信。号码18817566090    谢谢

**Comment by Espergium** [comment]
1.7.10 is discontinued. There are no updates.

**Comment by Ngar1** [comment]
Yes, I know and had been written. The question is, what would motivate to do it?
